node/src/chain_spec.rs

```diff
@@ -92,7 +92,7 @@ pub fn default_parachain_properties() -> Option<Properties> {
 	parachain_properties("LIT", 12, 31)
 }
 
-const DEFAULT_CANDIDACY_BOND: Balance = 16 * UNIT;
+const DEFAULT_CANDIDACY_BOND: Balance = 1;
```

Comment:
shuold not modify runtime

Reply:
test code from upstream, I discussed with Kailai, the constant just impacts the dev environemnt.